main/src/ca/uwaterloo/flix/api/lsp/provider/SymbolProvider.scala

```diff
@@ -0,0 +1,20 @@
+package ca.uwaterloo.flix.api.lsp.provider
+
+import ca.uwaterloo.flix.api.lsp.{DocumentSymbol, Index, Position, Range, SymbolKind, SymbolTag}
+import ca.uwaterloo.flix.language.ast.TypedAst.Root
+import ca.uwaterloo.flix.language.debug.{Audience}
+
+object SymbolProvider {
+
+  private implicit val audience: Audience = Audience.External
+
+  def processSymbols(uri: String)(implicit index: Index, root: Root): List[DocumentSymbol] =
+    List(DocumentSymbol("name",
+      Some("description"),
+      SymbolKind.Interface,
+      Range(Position(1, 1), Position(1, 5)),
+      Range(Position(1, 1), Position(1, 5)),
+      List(SymbolTag.Deprecated),
```

Comment:
For testing, it might be useful if the symbol also spans more than one line. Just a thought!

Reply:
Good point!

Reply:
I'll add also a child for testing purposes